modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/display/view.jsp

```diff
@@ -128,9 +128,17 @@ long formInstanceId = ddmFormDisplayContext.getFormInstanceId();
 
 							<%
 							DDMFormValuesValidationException.MustSetValidValue msvv = (DDMFormValuesValidationException.MustSetValidValue)errorException;
+
+							String fieldName = msvv.getFieldName();
+
+							LocalizedValue fieldLabel = msvv.getFieldLabel();
+
+							if (fieldLabel != null) {
+								fieldName = fieldLabel.getString(themeDisplay.getLocale());
```

Comment:
@robertoDiaz I was wondering if we could check if `fieldLabel.getString(themeDisplay.getLocale())` is empty and then use `fieldName` if it is. What do you think?

Reply:
sounds good. 👍 

Anyway, what do you think about generify this? (apply it to MustSetValidValuesSize and more too)

Reply:
I do not see a problem. How many would it be?

Reply:
Most of the DDMFormValuesValidationException inner exceptions (the ones that follow the same pattern as MustSetValidValue and receive the fieldName).

I'll do this changes and send it to you in separate commits.

Thanks!!